docker/wiremock/stubs/__files/correctPatientStructuredRecordWithTextDocAttachment.json

```diff
@@ -0,0 +1,1209 @@
+{
```

Comment:
Were these stubs used for testing within the PTL environment?

Reply:
yes